pom.xml

```diff
@@ -20,6 +20,34 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
Junit is included in spring boot test dependency

Reply:
removed